configure

```diff
@@ -7,6 +7,7 @@
 # pyenv will alert which shims are available and then will fail the build.
 _=[ 'exec' '/bin/sh' '-c' '''
 test ${TRAVIS} && exec python "$0" "$@"  # workaround for pyenv on Travis CI
+which python3.8 >/dev/null && exec python3.8 "$0" "$@"
```

Comment:
Should we maybe check for `python3` first?

Reply:
The trouble with doing that is that `python3` might be `in ((3, 0), (3, 1), (3, 2), (3, 3), (3, 4))`.